src/Components/AppShell/BugModal.scss

```diff
@@ -0,0 +1,16 @@
+.BugModal {
+  .Bug {
+    textarea {
+      min-height: 5rem; /* Overwrite Modal textarea styles */
```

Comment:
If this is meant to overwrite Modal textarea styles, you might consider "reopening" the `.Modal` class to make it more explicit.

```
.Modal {
  &.BugModal {
    // ... 
  }
}
```